src/history/HistoryArchive.h

```diff
@@ -64,12 +64,14 @@ struct HistoryArchiveState
 
     unsigned version{HISTORY_ARCHIVE_STATE_VERSION};
     std::string server;
+    std::string networkPassphrase;
```

Comment:
Now that we're including the network passphrase, should we have a sanity check when processing HAS files in catchup?

Reply:
Done!